test/application.py

```diff
@@ -58,17 +60,40 @@ def test_reloader(self):
         time.sleep(1)
         write('foo.py', data % dict(classname='c', output='c'))
         self.assertEquals(app.request('/').data, b'c')
```

Comment:
__self.assertEquals()__ (with the trailing '__s__') was removed in Python 3 so please use __self.assertEqual()__ instead.